release/models/ospf/openconfig-ospfv2-area-interface.yang

```diff
@@ -227,6 +233,55 @@ submodule openconfig-ospfv2-area-interface {
     }
   }
 
+  grouping ospfv2-area-interface-authentication-config {
+    description
+      "Configuration parameters relating to OSPF interface
+      authentication";
+
+    leaf auth-mode {
+      type identityref {
+        base oc-ospf-types:AUTH_MODE;
+      }
+      description
+        "Authentication mode to be used for OSPF interface links.";
+    }
+
+    leaf simple-password {
+      type string;
+      description
+        "Password for simple authentication mode to be specified for OSPF.";
+    }
+  }
+
+  grouping ospfv2-area-interface-message-digest-config {
+    description
+      "Configuration parameters relating to OSPF message-digest
+      authentication";
+
+    leaf md-key-id {
```

Comment:
No need to prefix these leafs w/ `md-` - already children of `message-digest` list